src/patternfly/components/BackgroundImage/background-image.scss

```diff
@@ -1,53 +1,63 @@
 @import "../../patternfly-utilities";
 
 .pf-c-background-image {
+  --pf-c-background-image--BackgroundImageFilter: url("#{$pf-global--image-path}/background-filter.svg#image_overlay");
+  --pf-c-background-image--BackgroundImage--xs: url("#{$pf-global--image-path}/pfbg_576.jpg");
+  --pf-c-background-image--BackgroundImage--xs-2x: url("#{$pf-global--image-path}/pfbg_576@2x.jpg");
+  --pf-c-background-image--BackgroundImage--sm: url("#{$pf-global--image-path}/pfbg_768.jpg");
+  --pf-c-background-image--BackgroundImage--sm-2x: url("#{$pf-global--image-path}/pfbg_768@2x.jpg");
+  --pf-c-background-image--BackgroundImage--md: url("#{$pf-global--image-path}/pfbg_992.jpg");
+  --pf-c-background-image--BackgroundImage--md-2x: url("#{$pf-global--image-path}/pfbg_992@2x.jpg");
+  --pf-c-background-image--BackgroundImage--lg: url("#{$pf-global--image-path}/pfbg_1200.jpg");
+  --pf-c-background-image--BackgroundImage--xl: url("#{$pf-global--image-path}/pfbg_2000.jpg");
+
   position: fixed;
   top: 0;
   left: 0;
   z-index: -1;
   width: 100%;
   height: 100%;
   background-color: var(--pf-global--BackgroundColor--dark-200);
-  background-image: url("#{$pf-global--image-path}/pfbg_576.jpg");
-  filter: url("#{$pf-global--image-path}/background-filter.svg#image_overlay");
+  background-image: var(--pf-c-background-image--BackgroundImage--xs);
+  filter: var(--pf-c-background-image--BackgroundImageFilter);
```

Comment:
The syntax follows `selector--Propierty` in this case it shold be:

`--pf-c-background-image--Filter`

Reply:
Ok. I renamed that filter variable.